Sources/Widgets/Representations/WidgetRepresentation/index.js

```diff
@@ -161,6 +161,42 @@ function vtkWidgetRepresentation(publicAPI, model) {
     }
   };
 
+  function applyCoincidentTopologyParametersToMapper(
+    mapper,
+    { factor, offset }
+  ) {
+    mapper.setResolveCoincidentTopologyToPolygonOffset();
+    mapper.setRelativeCoincidentTopologyPolygonOffsetParameters(factor, offset);
+    mapper.setRelativeCoincidentTopologyLineOffsetParameters(factor, offset);
+    mapper.setRelativeCoincidentTopologyPointOffsetParameters(factor, offset);
+  }
+
+  // Add warning to model.actors.push
+  model.actors.push = (...args) => {
+    console.warn(
+      'You should use publicAPI.addActor() to initialize the actor properly'
+    );
```

Comment:
@jourdain is it a best-practice in VTK-JS ?

Reply:
@floryst

Reply:
I would use `vtkWarningMacro` here.

Reply:
You could also invoke `publicAPI.addActor` here as well, unless you want to leave `model.actors.push` as a means of adding an actor without applying the topology parameters.